host-configs/rzansel-blueos_3_ppc64le_ib_p9-clang@10.0.1.2_cuda.cmake

```diff
@@ -83,13 +83,13 @@ set(CUDA_TOOLKIT_ROOT_DIR "/usr/tce/packages/cuda/cuda-11.2.0" CACHE PATH "")
 
 set(CMAKE_CUDA_ARCHITECTURES "70" CACHE STRING "")
 
+set(CMAKE_CUDA_FLAGS "" CACHE STRING "")
+
 set(ENABLE_CUDA ON CACHE BOOL "")
 
 set(CMAKE_CUDA_SEPARABLE_COMPILATION ON CACHE BOOL "")
 
-set(CMAKE_CUDA_ARCHITECTURES "70" CACHE STRING "")
-
-set(CMAKE_CUDA_FLAGS "-restrict --expt-extended-lambda " CACHE STRING "")
+set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -restrict --expt-extended-lambda " CACHE STRING "" FORCE)
```

Comment:
Do we know why there are now two lines for `CMAKE_CUDA_FLAGS`?

I'm pretty sure the second `CMAKE_CUDA_FLAGS` w/ `FORCE` will overwrite the first one, so it's not a problem, but it does seem a bit weird to have both.

Reply:
I have no idea why there are two lines for `CMAKE_CUDA_FLAGS`. I just ran the scripts to generate the new host-configs.

Reply:
It's likely related to the class hierarchy for packages in Spack.
Thoughts @white238 ?

Reply:
There were some changes in spack. This was reviewed in a previous PR.

Reply:
A note about no longer supporting XL has been added to the release notes.